src/stores/store.ts

```diff
@@ -321,16 +321,26 @@ export const useStore = defineStore('store', {
         await client.startFarming(this.plotPath, this.plotSizeGB);
 
         // TODO: consider moving logging to client.ts
-        util.infoLogger('PLOTTING PROGRESS | farmer started');
+        util.infoLogger('farmer started');
 
+      } catch (error) {
```

Comment:
I don't really get why adding one more `try-catch` if error handling is the same for both